flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/service/FlinkService.java

```diff
@@ -401,12 +400,6 @@ public Optional<String> cancelSessionJob(
         return savepointOpt;
     }
 
-    public void stopSessionCluster(
```

Comment:
After you rebase the latest master, I am afraid that we still need the `stopSessionCluster`, which simply call the `FlinkUtils.deleteCluster` in the production code. And we could still keep the testing implementation.

Reply:
`stopSessionCluster` is back now